enarx-merge

```diff
@@ -6,18 +6,39 @@ import enarxbot
 # Initialize Github
 github = enarxbot.connect()
 
-# Print column keys.
-print(" Merged? | Repository + PR number | Mergeable? | State  ")
-
+# Timestamp all PRs.
+prs_timestamped = {}
 for issue in github.search_issues("org:enarx state:open is:public is:pr review:approved"):
-    # Status strings.
-    merged_indicator = "✗"
+    # Construct a PR identifier before conversion.
     pr_identifier = f"{issue.repository.name}#{issue.number}"
 
     # `search_issues()` returns issues. Convert to PRs.
     pr = issue.as_pull_request()
 
-    # If all merge criteria pass, merge the PR.
+    # Get the latest PR review
+    latest_review = ""
+    for i in pr.get_reviews():
+        if i.submitted_at.isoformat() > latest_review:
+            latest_review = i.submitted_at.isoformat()
+
+    prs_timestamped[latest_review] = (pr, pr_identifier)
```

Comment:
```suggestion
    prs_timestamped[latest_review] = (issue, pr)
```
Then you can construct the `pr_identifier` later while also having the full issue object for later script modifications.